src/lineChunked.js

```diff
@@ -430,27 +427,29 @@ export default function () {
     const chunks = getChunkNames();
 
     // for each chunk, evaluate the attrs and styles to use for lines and points
-    chunks.forEach(chunkName => {
+    chunks.forEach((chunkName) => {
       const chunkDef = chunkDefinitions[chunkName] || {};
       const evaluatedChunk = {
-        styles: Object.assign({},
```

Comment:
Why bother changing this to not use Object.assign?

Reply:
Again, no personal preference, but this is eslint enforced.